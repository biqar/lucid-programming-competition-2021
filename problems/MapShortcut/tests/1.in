1
Left 45 10
