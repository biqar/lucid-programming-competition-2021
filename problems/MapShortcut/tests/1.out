Left 45 10
